docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: User Guides
+description: Documentation for end users
+h1: End User Documentation
+---
+
+Teleport is a complex security solution, and most of our documentation covers
```

Comment:
I don't think we need this first sentence.

Reply:
Removed and added more context.